sdk/spring/scripts/compatibility_insert_dependencymanagement.py

```diff
@@ -60,10 +60,15 @@ def get_repo_position(pom_file_content):
 def get_repo_content_without_tag():
     return """
     <repository>
-      <id>repository.springframework.maven.milestone</id>
-      <name>Spring Framework Maven Milestone Repository</name>
+      <id>repository.springframework.maven.snapshot</id>
+      <name>Spring Framework Maven SNAPSHOT Repository</name>
       <url>https://repo.spring.io/snapshot/</url>
     </repository>
+    <repository>
+      <id>repository.springframework.maven.milestone</id>
+      <name>Spring Framework Maven Milestone Repository</name>
+      <url>https://repo.spring.io/milestone/</url>
+   </repository>
```

Comment:
should we separate these two into two methods? One for snapshot, one for milestone?